app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/sap_profile.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::SAPProfile < ::Flavor
+end
```

Comment:
Don't forget to `require_nested :SAPProfile` in the `PowerVirtualServers::CloudManager`